src/components/Stack/tests/Stack.test.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Stack from '../Stack';
+import Badge from '../';
+import Item from '../Item';
+import Heading from '../';
+
+describe('Stack Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined', () => {
+            const stackWrapper = mount(
+                                        <Stack>
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.childAt(0).prop('children').props.children).toBe('Paid');
+            expect(stackWrapper.childAt(1).prop('children').props.children).toBe('Fulfilled');
+
+        });
+
+         it('should verify children property is not defined', () => {
+            const stackWrapper = mount(<Stack/>);
+            expect(stackWrapper.prop('children')).toBeUndefined();
+        });
+    });
+
+    describe('Test vertical property', () => {
+        it('should verify vertical property is set', () => {
+            const stackWrapper = mount(
+                                        <Stack vertical>
+                                          <Badge>Paid</Badge>
```

Comment:
Why do you keep using the Badge component, but never actually test it? All your tests would work without line 32 and 33. Doesn't break anything, but seems redundant.